cmake/compileroptions.cmake

```diff
@@ -33,8 +33,8 @@ if (USE_ANALYZE)
 endif()
 
 set(CMAKE_CXX_FLAGS_ASAN "-g -fsanitize=address,undefined -fno-sanitize-recover=all"
-    CACHE STRING "Compiler flags in asan build"
-    FORCE)
+        CACHE STRING "Compiler flags in asan build"
```

Comment:
I am no cmake guy. but why did you change this indentation? the old indentation seems logical to me.

Reply:
I did not change this intentionally - it was done automatically by my IDE and somehow some of these changes do not show up in the Diffs of it...will try to revert them.